src/wp-includes/script-loader.php

```diff
@@ -1843,6 +1843,26 @@ function wp_just_in_time_script_localization() {
 	);
 }
 
+
+/**
+ * Prints a loader script if there is text/template registered script.
+ */
```

Comment:
Per [PHP Documentation Standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods), Use proper format for function

Reply:
Updated. Usage of `since` is still missing as this will be handled together for the entire functionality in a separate task.  Thanks @mukeshpanchal27.